provider/gcp/config.go

```diff
@@ -24,23 +24,43 @@ import (
 )
 
 const (
+	AMD64 = "x86_64"
+	ARM64 = "arm64"
```

Comment:
we might want to make these consts part of the API specs to make it clear of supported values